modules/engines/engine-hdb/src/test/java/com/sap/xsk/hdb/ds/test/itest/hdbview/XSKHDBViewParserHanaCase.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.hdb.ds.test.itest.hdbview;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import com.google.inject.name.Names;
+import com.sap.xsk.hdb.ds.api.XSKDataStructuresException;
+import com.sap.xsk.hdb.ds.facade.IXSKHDBCoreFacade;
+import com.sap.xsk.hdb.ds.test.itest.model.JDBCModel;
+import com.sap.xsk.hdb.ds.test.itest.module.XSKHDBTestModule;
+import com.sap.xsk.utils.XSKHDBUtils;
+import org.eclipse.dirigible.commons.config.Configuration;
+import org.eclipse.dirigible.core.scheduler.api.SynchronizationException;
+import org.eclipse.dirigible.repository.local.LocalResource;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static com.sap.xsk.hdb.ds.test.itest.utils.TestConstants.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class XSKHDBViewParserHanaCase {
+
+  private static Connection connection;
+  private static IXSKHDBCoreFacade facade;
+
+  @BeforeClass
+  public static void setUpBeforeClass() throws SQLException, IOException {
+    JDBCModel model = new JDBCModel(HANA_DRIVER, Configuration.get("db.url"), Configuration.get("db.username"),
+        Configuration.get("db.password"));
+    Injector injector = Guice.createInjector(new XSKHDBTestModule(model));
+    connection = injector.getInstance(DataSource.class).getConnection();
+    facade = injector.getInstance(Key.get(IXSKHDBCoreFacade.class, Names.named("xskHDBCoreFacade")));
+  }
+
+  @After
+  public void cleanUpAfterTest() throws SQLException {
```

Comment:
I think the order of the `@After` and `@Before` methods is changed.

Reply:
Removed @After clause. Create/Drop statements for additional tables is now moved in the test itself.